.github/workflows/release.yaml

```diff
@@ -79,6 +79,35 @@ jobs:
           draft: true
           prerelease: false
 
+      - name: Create the PR to bump the version in the main branch (only if we're tagging from main branch)
+        if: ${{ github.event.inputs.branch == 'main' }}
+        run: |
```

Comment:
Is this bash from podman-desktop?

Reply:
Not sure WDYM ?

Reply:
It was inspired by Podman Desktop and bootc extension